controls/def.cf

```diff
@@ -196,6 +196,14 @@ bundle common def
       comment => "Define plugins path",
       handle => "common_def_vars_dir_plugins";
 
+      "server_controls[dir_templates]"
+        string => ifelse( isvariable( "def.server_controls[dir_templates]"),
+                          $(def.server_controls[dir_templates]),
+                          "$(sys.workdir)/templates"),
```

Comment:
Since the MPF ships with a template directory in the root do you think that would be a more appropriate default?